### DISCUSSION TITLE
How to change status?

### DISCUSSION THREAD

**Discussion by Aryan700coder** [danthe1st/Mee6-bypasser#54] [Questions] [answered]
I am new in java I wanted to ask how we can change status of the bot?
I made the bot from source code.

**Comment by danthe1st** [comment, marked as answer]
You just need to edit [Line 37/38 of `MeeBypasser.java`](https://github.com/JDiscordBots/Me-e--bypasser/blob/e3253daac7f94bddb2e2c36e706b68048da474f1/src/main/java/io/github/jdiscordbots/mee/bypasser/MeeBypasser.java#L37) before building.

Line 37 sets the online status and line 38 sets the activity.
